enhancements/scorecard-storage.md

```diff
@@ -0,0 +1,333 @@
+```bash
+./test-output
+./test-output/custom-suite/example-test1/somefile
+./test-output/custom-suite/example-test2/anotherfile
+./test-output/custom-suite/example-test3/foo.log
+```
+This directory will be over-written if scorecard is executed multiple times, it is therefore up to the end user
+to perform data management of this test output directory as they require.
+
+### Test Plan
+
+**Note:** *Section not required until targeted at a release.*
+
+Consider the following in developing a test plan for this enhancement:
+- Will there be e2e and integration tests, in addition to unit tests?
+- How will it be tested in isolation vs with other components?
+
+No need to outline all of the test cases, just the general strategy. Anything
+that would count as tricky in the implementation and anything particularly
+challenging to test should be called out.
+
+All code is expected to have adequate tests (eventually with coverage
+expectations).
+
+### Graduation Criteria
+
+**Note:** *Section not required until targeted at a release.*
+
+Define graduation milestones.
+
+These may be defined in terms of API maturity, or as something else. Initial proposal
+should keep this high-level with a focus on what signals will be looked at to
+determine graduation.
+
+Consider the following in developing the graduation criteria for this
+enhancement:
+- Maturity levels - `Dev Preview`, `Tech Preview`, `GA`
+- Deprecation
+
+Clearly define what graduation means.
+
+#### Examples
+
+These are generalized examples to consider, in addition to the aforementioned
+[maturity levels][maturity-levels].
+
+##### Dev Preview -> Tech Preview
+
+- Ability to utilize the enhancement end to end
+- End user documentation, relative API stability
+- Sufficient test coverage
+- Gather feedback from users rather than just developers
+
+##### Tech Preview -> GA 
+
+- More testing (upgrade, downgrade, scale)
+- Sufficient time for feedback
+- Available by default
+
+**For non-optional features moving to GA, the graduation criteria must include
+end to end tests.**
+
+
+
+### Upgrade / Downgrade Strategy
+
+This feature doesn't impact existing scorecard tests or test configurations.
+
+I currently don't see value in backporting this feature to previous versions of the operator-sdk.
+
+### Version Skew Strategy
+
+
+## Implementation History
+
+
+
+## Drawbacks
+
+This feature and suggested implementation depend on dynamic storage
+being available on-cluster.
+
+
+## Alternatives
+
+### Alternative 1
+An alternative to this feature would be to design a means of embedding test content into the test Pod log itself, co-mingled with the current scorecard test result JSON.
+
+This seems really complicated and hard to maintain so I rejected this in favor of this proposal.
+
+### Alternative 2
+
+Another alternative to creating PVCs for test output storage, you could:
+ * create a sidecar container in the test pod
```

Comment:
could also be done w/ init containers:

initcontainer1:  runs tests
main container: sticks around until scorecard tells it it's done

Reply:
possibly but scorecard already uses an initContainer to handle mapping and mounting of bundle contents, so the design/implementation/lifecycle mgmt would possibly get messy.

Reply:
a series of init containers doesn't seem to complicated.  it's what builds + tekton do.